eng/pipelines/runtime.yml

```diff
@@ -250,6 +250,62 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_mono.containsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+#
+# Build Mono LLVM debug
+# Only when libraries or mono changed
```

Comment:
Shouldn't this also build when llvm dependency is updated?

Reply:
I... have no idea how to represent that.

Reply:
@ViktorHofer can you help?

Reply:
It's already doing that. Everything's included by default + the additional exclude minus the exclude: https://github.com/dotnet/runtime/blob/master/eng/pipelines/runtime.yml#L74-L89.